packages/react-app/src/components/Container.jsx

```diff
@@ -45,12 +46,45 @@ const Container = () => {
       >
         <AvailableLiquidity />
       </Box>
-      <Image
-        src="https://storage.googleapis.com/1hive/xpollinate/powered-by.svg"
-        bottom="0"
-        h="2rem"
-        marginTop="calc(5% + 60px)"
-      />
+      <div style={{
+        display: "flex",
+       flexDirection: isLargerThan1280 ? "row" : "column",
+        marginTop: "calc(5% + 60px)",
+       alignItems: "center",
+        justifyContent: "center"
+      }}>
+        <Image
+          src={require("../assets/poweredby.svg")}
+          bottom="0"
+         height="1.3rem"
+         width="10.5rem"
+         marginLeft= {isLargerThan1280 ? "0rem" : "0.6rem"}
+         marginTop= {isLargerThan1280 ? "0.2rem" : "0rem"}
+
+       />
+        <a href="https://connext.network/">
+         <Image
+           src={require("../assets/Connext.svg")}
+           bottom="0"
+           height="2.5rem"
+           width="10rem"
+           marginRight= {isLargerThan1280 ? "0.2rem" : "0rem"}
+
+         />
+       </a>
+       <h2 style={{marginLeft: "15px"}}>x</h2>
+       <a href="https://1hive.org">
+         <Image
+           src={require("../assets/1Hive.svg")}
+           bottom="0"
+           height="2.5rem"
+           width="7.6rem"
+           marginLeft= {isLargerThan1280 ? "1.5rem" : "0rem"}
+           marginBottom={isLargerThan1280 ? "0.5rem" : "0rem"}
+
+          />
+       </a>
```

Comment:
The issue with the spacing is the `width` prop on the "Powered By" image. I've added some other adjustments, such as maintaing the correct aspect ratio on images, adjusting some sizes and so on. Let me know what you think.

```suggestion
         <Image
          src={require("../assets/poweredby.svg")}
         height="1.3rem"
          marginRight={isLargerThan1280 ? "0.5rem" : "0rem"}
         marginLeft={isLargerThan1280 ? "0rem" : "0.6rem"}
         marginTop= {isLargerThan1280 ? "0.2rem" : "0rem"}
       />
        <a href="https://connext.network/">
         <Image
           src={require("../assets/connext.svg")}
           height="2.5rem"
           marginRight={isLargerThan1280 ? "0.2rem" : "0rem"}
         />
       </a>
       <h2 style={{marginLeft: "15px"}}>x</h2>
       <a href="https://1hive.org">
         <Image
           src={require("../assets/1hive.svg")}
           height="2rem"
           marginBottom="0.5rem"
           marginLeft={isLargerThan1280 ? "1.5rem" : "0rem"}
          />
       </a>
```

Reply:
Looks great! Yes I already knew images are case-sensitive, I noticed it was different at the time of the push in GitHub, but in my laptop it was working properly (I think the images  maybe didn't upload properly, sorry about it). 
Just added your suggestions (made a commit from the code, have never tried the "Commit suggestion" button from GitHub, just realized it exists)
Thanks for the help man! Please lmk if it works properly or if I should continue doing something.